internal/app/mistapiconnector/mistapiconnector_app.go

```diff
@@ -1126,13 +1126,13 @@ func (mc *mac) SetupTriggers(ownURI string) error {
 		apiURL := mc.lapi.GetServer() + "/api/stream/" + mc.baseStreamName
 		presets := []string{"P144p30fps16x9"}
 		// base stream created with audio disabled
```

Comment:
Oooohhh could this be the root cause for those audio-less recordings we've seen a couple of times?